Experimental/DedicatedGameServer/Assets/Plugins/ParrelSync/ScriptableObjects/ParrelSyncProjectSettings.asset

```diff
@@ -0,0 +1,15 @@
+%YAML 1.1
```

Comment:
Not sure what this is, it's missing a reference to its script, so this could be removed entirely too?

Reply:
Looks like some leftover from LP's first iterations with ParrelSync. Removed!